crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -139,6 +139,11 @@ def bulk
               dirty = true
             end
 
+            unless node.default_fs == node_attributes["default_fs"]
```

Comment:
Hence unnecessary. Anyway, it doesn't hurt here, does it? I mean, it would be useful to have it also in bulk view, so once the UI manages to fit everything we want into the page, this part is ready